web_src/less/_base.less

```diff
@@ -73,6 +73,22 @@
   --color-black: #1b1c1d;
   --color-gold: #a1882b;
   --color-white: #ffffff;
+  --color-green-ok-bg: #fcfff5;
+  --color-green-ok-border: #a3c293;
+  --color-green-ok-header: #1a531b;
+  --color-green-ok-text: #2c662d;
+  --color-red-err-bg: #fff6f6;
+  --color-red-err-border: #e0b4b4;
+  --color-red-err-header: #912d2b;
+  --color-red-err-text: #9f3a38;
+  --color-red-sha-bg: rgba(219, 40, 40, .1);
+  --color-red-sha-hover: rgba(219, 40, 40, .3);
+  --color-orange-sha-bg: rgba(242, 113, 28, .1);
+  --color-orange-sha-hover: rgba(242, 113, 28, .3);
+  --color-yellow-sha-bg: rgba(251, 189, 8, .1);
+  --color-yellow-sha-hover: rgba(251, 189, 8, .3);
+  --color-green-sha-bg: rgba(33, 186, 69, .1);
+  --color-green-sha-hover: rgba(33, 186, 69, .3);
```

Comment:
1. Could you convert to `#rrggbbaa` syntax? Just for consistency basically.
2. Can we use `error` (red), `warning` (orange), `info` (blue) and `success` (green) states please?
2. Based on previous point, please remove color names from these variable names, e.g. `--color-red-err-bg` -> `--color-error-bg`, `--color-green-ok-bg` -> `--color-success-bg`.